Core/Src/compute.c

```diff
@@ -516,12 +516,17 @@ void compute_send_segment_temp_message(acc_data_t* bmsdata)
         int8_t segment2_average_temp;
         int8_t segment3_average_temp;
         int8_t segment4_average_temp;
+		int8_t segment5_average_temp;
+		int8_t segment6_average_temp;
+
     } segment_temp_msg_data;
 
     segment_temp_msg_data.segment1_average_temp = bmsdata->segment_average_temps[0];
     segment_temp_msg_data.segment2_average_temp = bmsdata->segment_average_temps[1];
     segment_temp_msg_data.segment3_average_temp = bmsdata->segment_average_temps[2];
     segment_temp_msg_data.segment4_average_temp = bmsdata->segment_average_temps[3];
+	segment_temp_msg_data.segment5_average_temp = bmsdata->segment_average_temps[4];
+	segment_temp_msg_data.segment6_average_temp = bmsdata->segment_average_temps[5];
 
     uint8_t buff[4] = { 0 };
```

Comment:
whats buf used for here?

Reply:
whoops. look like when that was cleaned up by cangen, that was leftover. will remove